src/js/controllers/walletDetails.js

```diff
@@ -185,9 +231,52 @@ angular.module('copayApp.controllers').controller('walletDetailsController', fun
     });
   };
 
+  $scope.backup = function() {
+   //$state.go('tabs.preferences', {walletId: $scope.walletId});
+   //$state.transitionTo('tabs.preferences.backupWarning');
+  };
+
+  $scope.amountHeight = '180px';
+  $scope.amountScale = 'scale3d(' + 1 + ',' + 1 + ',' + 1+ ')';
```

Comment:
What about creating a new class with this ?

>.amountScale {
    transform: scale3d(' + 1 + ',' + 1 + ',' + 1+ ');
}

And then use _ng-class_ instead of _ng-style_ (not tested)